drip/drips.py

```diff
@@ -20,25 +25,69 @@
 
 import logging
 
+DEFAULT_DRIP_MESSAGE_CLASS = 'drip.drips.DripMessage'
 
-def configured_message_classes() -> dict:
-    """[summary]
 
-    :return: [description]
-    :rtype: dict
+def configured_message_classes() -> ChainMap:
+    """
+    Returns a ChainMap (basically a dict), between default settings and
+    settings.DRIP_MESSAGE_CLASSES which should be a dictionary of the form:
+
+        { message_class_name: class_path }
+
+    For example, if you define this at DRIP_MESSAGE_CLASSES:
+
+        {
+            'default': 'my.path.to.DripMessage',
+            'my_other_class': 'my.other.path.to.DripMessage',
+        }
+
+    The returning value of this method would be:
+
+        {
+            'default': 'my.path.to.Drip',
+            'my_other_class': 'my.other.path.to.DripMessage',
+        }
+
+    But, if you define this:
+
+        {
+            'my_other_class': 'my.other.path.to.DripMessage',
+        }
+
+    The returning value of this method would be:
+
+        {
+            'default': 'drip.drips.DripMessage',
+            'my_other_class': 'my.other.path.to.DripMessage',
+        }
+
+    A 'default' key is added when it's not present on DRIP_MESSAGE_CLASSES.
```

Comment:
What happens if the users doesn't define a `default` class, and the `my_other_class` is `drip.drips.DripMessage`?
Is that case possible? Is it covered?

Reply:
```python
    # We set a default configuration dict.
    default_config = {'default': DEFAULT_DRIP_MESSAGE_CLASS}
    # Tries to get a user-defined config, falling back to an empty dict if config is not found.
    user_defined_config = getattr(settings, 'DRIP_MESSAGE_CLASSES', {})
    # Chain-map those dictionaries
    return ChainMap(user_defined_config, default_config)
```